osc-rest-server/src/main/java/org/osc/core/broker/rest/server/LogComponent.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.rest.server;
+
+import static org.osgi.service.component.annotations.ReferencePolicyOption.GREEDY;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.slf4j.ILoggerFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component
+public class LogComponent {
```

Comment:
Maybe name it LogProvider? Also should this really be in the osc-rest-server project? I thought it would be in the osc-server project. Maybe in the pkg osc.core.broker.util? @arvindn05  thoughts?

Reply:
Emanoel:  I thought we agreed on LogComponent in each bundle.

Reply:
I recall there were two different kind of LogComponents (I think provider is a better name). A simple one in various bundles that would just act as a simple factory. A more complex one in the server that allow for the proper server start up with a dynamic dependency.

Reply:
1.  I renamed to LogProvider, but didn't push tat yet
2. Yes, those changes are much simpler. Those changes aren't out yet, either.